internal/saucecloud/cloud.go

```diff
@@ -311,7 +314,41 @@ func (r *CloudRunner) logSuiteConsole(res result) {
 	// Display log only when at least it has started
 	assetContent, err := r.JobReader.GetJobAssetFileContent(context.Background(), res.job.ID, ConsoleLogAsset)
 	if err != nil {
-		log.Warn().Str("suite", res.suiteName).Msg("Failed to retrieve the console output.")
+		assetContent, err = r.JobReader.GetJobAssetFileContent(context.Background(), res.job.ID, "junit.xml")
+		if err == nil {
```

Comment:
Rather than nesting it further, could we exit earlier instead? Just a lot if nested ifs now, making it hard to parse visually which block belongs where 😅

Reply:
👍 To your point, I confused myself while cleaning it up. 😆